lightwood/api/high_level.py

```diff
@@ -17,6 +17,18 @@
 import gc
 import time
 from lightwood.helpers.log import log
+from shutil import copyfile
+
+
+def load_custom_module(file_path: str):
+    modules_dir = os.path.join(os.path.expanduser('~/lightwood_modules'))
+    custom_module_path = os.path.join(modules_dir, os.path.split(file_path)[-1])
+    if not os.path.exists(modules_dir):
+        os.mkdir(modules_dir)
+    if os.path.exists(custom_module_path):
+        os.remove(custom_module_path)
```

Comment:
I'm not sure if removing is the best idea

Reply:
Module loading based on the package version is out of scope for this PR, I assume?

Reply:
Also +1 to removing being too harsh. Maybe move contents to `custom_module_path/old` or something like that.